File: Euler7.java
public class Euler7
{
	public static void main(String[] args) {
	    int numPrimes = 1;
        int numm = 1;
 
        while (numPrimes < 10001) {
            numm = numm + 2;
            if (checkPrime(numm)) {
                numPrimes++;
                
            }
        }   
        System.out.println(numm);
	}
	
	static boolean checkPrime(long num){
	    if(num > 2){
			for(long i = 2; i <= Math.sqrt(num); i++){
				if(num % i == 0)
					return false;
			}
			return true;
		} else
			return true;
	}
}

File: Python/Euler7.py
import math
def check_prime(num):
    if(num > 2):
        for i in range(2, int(math.sqrt(num) + 1)):
            if(num%i==0):
                return False
        return True
    else:
        return True

numPrimes = 1
num = 1
while (numPrimes<10001):
    num = num + 2
    if(check_prime(num)):
        numPrimes = numPrimes + 1
    
print(num)